cronjobs/bsz_util.py

```diff
@@ -82,18 +94,18 @@ def GetBackupDirectoryPath(config):
 
 # Check whether all the instances are needed
 def NeedsBothInstances(filename_regex):
-     without_localdata_pattern = "_o[)]?-[(]?.*[)]?"
-     without_localdata_regex = re.compile(without_localdata_pattern)
-     return without_localdata_regex.search(filename_regex.pattern) is not None
+    without_localdata_pattern = "_o[)]?-[(]?.*[)]?"
+    without_localdata_regex = re.compile(without_localdata_pattern)
+    return without_localdata_regex.search(filename_regex.pattern) is not None
 
 
 # For IxTheo our setup requires that we obtain both the files with and without local data because otherwise
 # we get data inconsistencies
 def AreBothInstancesPresent(filename_regex, remote_files):
-     if not remote_files:
+    if not remote_files:
         return True
-     matching_remote_files = list(filter(filename_regex.match, remote_files))
-     return True if len(matching_remote_files) % 2 == 0 else False
+    matching_remote_files = list(filter(filename_regex.match, remote_files))
+    return True if len(matching_remote_files) % 2 == 0 else False
```

Comment:
Arrgh!  Delete "True if" and "else False" from this line!

Reply:
done